Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageListView.swift

```diff
@@ -241,3 +249,28 @@ public struct DateIndicatorView: View {
         }
     }
 }
+
+struct TypingIndicatorBottomView: View {
+    @Injected(\.colors) private var colors
+    @Injected(\.fonts) private var fonts
+    
+    var typingIndicatorString: String
+    
+    var body: some View {
+        VStack {
+            Spacer()
+            HStack {
+                TypingIndicatorView()
+                Text(typingIndicatorString)
+                    .font(.footnote)
+                    .foregroundColor(Color(colors.textLowEmphasis))
+                Spacer()
```

Comment:
I was always annoyed by the solution to use a `Spacer` here as the basic goal is to make the text take up the remaining space, right?

So, just a suggestion because I found out about how to do it and liked the approach more. But it is a very subjective thing, we can also just leave it as is!

```suggestion
                    .foregroundColor(Color(colors.textLowEmphasis))
                    .frame(maxWidth: .infinity, alignment: .leading)
```

Reply:
In this case there's a difference - the spacer fills the remaining space, but keeps the text view with its own size. The frame maxWidth, increases the text view size to the max available width and in this case, we don't actually need that. We should avoid maxWidth - for example putting maxWidth in the link preview attachment in the past, made the scrolling if the message list very slow. I guess some more computing is needed to determine what would be the size in such cases.